flintrock/ec2.py

```diff
@@ -785,7 +785,7 @@ def _create_instances(
         if spot_requests:
             request_ids = [r['SpotInstanceRequestId'] for r in spot_requests]
             if any([r['State'] != 'active' for r in spot_requests]):
-                logger.info("Canceling spot instance requests...", file=sys.stderr)
+                print("Canceling spot instance requests...", file=sys.stderr)
```

Comment:
Because it doesn't work. When you call a log, you can't decide where it will go. You need to configure on log creation where the output will be (stdout, stderr, file or a combination of all this...)